src/Arcus.Templates.AzureFunctions.Http/Dockerfile

```diff
@@ -1,8 +1,8 @@
-FROM mcr.microsoft.com/azure-functions/dotnet:3.0 AS base
+FROM mcr.microsoft.com/azure-functions/dotnet:4 AS base
 WORKDIR /app
 EXPOSE 80
 
-FROM mcr.microsoft.com/dotnet/core/sdk:3.1-buster AS build
+FROM mcr.microsoft.com/dotnet/sdk:6.0.100-bullseye-slim AS build
```

Comment:
What is our strategy for selecting base images ? 
Buster & bullseye are both images for debian Linux.  I have no problem with that, but I just want to understand why we're using these images here, and for webapi for instance, we're using alpine.

Reply:
I think it's because of the compatibility with Azure Functions. I don't think we can use `alpine` here.

Reply:
I thought it would be something like that; unfortunately, I was not able to find any documentation on that.